### ISSUE TITLE
"Read more" link in network catalogue does not receive keyboard focus

### ISSUE THREAD

**Issue by MichSpieler** [pressbooks/pressbooks#4430]
<img width="1047" height="474" alt="Image" src="https://github.com/user-attachments/assets/8898802c-946e-4859-b4bb-5a321abaee44" />
